modules/gitops-in-built-permissions.adoc

```diff
@@ -5,7 +5,7 @@
 [id="in-built-permissions_{context}"]
 = In-built permissions for Argo CD
 
-This section lists the permissions that are granted to ArgoCD to manage specific cluster-scoped resources which include platform operators, optional OLM operators, and user management. Note that ArgoCD is not granted `cluster-admin` permissions.
+This section lists the permissions that are granted to ArgoCD to manage specific cluster-scoped resources which include cluster operators, optional OLM operators, and user management. Note that ArgoCD is not granted `cluster-admin` permissions.
```

Comment:
Should Operators e capitalized after cluster here?

Reply:
I left this mostly alone since it was outside of my normal sections, and they weren't capitalizing "Operator" in the rest of the section either. But I thought it was safe enough to at least change "platform" to "cluster" so it doesn't mess them up in the future.